front/app/component-library/components/Tooltip/index.tsx

```diff
@@ -62,6 +63,7 @@ const PLUGINS = [
       };
     },
   },
+  followCursor,
```

Comment:
Does this apply the behavior to all tooltips? I'm a little bit worried that this could introduce unexpected bugs in certain places since we use tooltips in so many different places. Would be less risk if this was behind a prop flag or something. But if you are confident this won't introduce any bugs in other places, we can keep it like this, because the behavior probably makes more sense like this anyway.